codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/structwithnestedblobtype.java

```diff
@@ -118,17 +111,17 @@ private BuilderImpl(StructWithNestedBlobType model) {
         }
 
         public final ByteBuffer getNestedBlob() {
-            return nestedBlob;
+            return nestedBlob == null ? null : nestedBlob.asByteBuffer();
         }
 
         @Override
-        public final Builder nestedBlob(ByteBuffer nestedBlob) {
+        public final Builder nestedBlob(SdkBytes nestedBlob) {
```

Comment:
Probably not related to this PR, but I think we can remove `final` from the methods since `BuilderImpl` is marked as `final`

Reply:
That would be fine with me.